doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -278,6 +278,7 @@ integration with 3rd party tools like logstash.
       enabled: yes
       filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
       filename: eve.json
+      version: 2 #1|2, 2 will enable new logging format (actually used only for dns logging)
```

Comment:
I've added some comments here, https://redmine.openinfosecfoundation.org/issues/2167 with respect to versioning. My worry about doing it here, is we'd have to up it to 3 when we break TLS, up it to 4 if we break http, and so on. So I think we need 2 levels of versioning. In this case "eve" would stay at version 1, and "dns" would move to version 2.